### PR TITLE
Release/6.8.0

### PR DESCRIPTION
**Pull request by kopijunkie**
## Added

- Internal: Add type defition for `borderRadiusSurfaceModal` customisation option.
- Public: Add information about Lokalise CLI v2 in CONTRIBUTING doc
- Internal: Refactor `useSdkOptions()` hook to return a tuple: `[options, { findStep }]`
- Internal: Added default filename for all document uploads if filename is not present.
- Public: Added user consent content reload screen
- Public: When `photoCaptureFallback` option is disabled in the face step, the requested variant is video and browser does not support MediaRecorder, attempt to redirect to the cross-device flow or display the unsupported browser error.
- Internal: Refactor for better reusability of video capture components.

### Changed

- Internal: Migrate CI build from TravisCI to Github Actions

### Fixed

- Public: Get latest copy from Lokalise with various grammar, punctuation fixes. Also reverts French, Spanish translations for some Proof of Address copy in these languages' locale files (Proof of Address is only supported for English)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +58,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Replaced some Sass variables with CSS variables to allow customization of colors and fonts.
 - Public: Added new enterprise feature `logoCobrand`. When purchased and enabled allows integrator to provide their own logo image to be displayed alongside the Onfido logo.
 - Internal: Use Node 14 LTS for Travis to be consistent with `.nvmrc` and `Dockerfile`.
+- Internal: Enable `strict` mode in tsconfig.json
```
Comment (stefaniacardenas) [inline CHANGELOG.md:61]: This entry doesn't look like it belongs to this version

File: MIGRATION.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 The guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `6.6.0` -> `[next-version]`
```
Comment (stefaniacardenas) [inline MIGRATION.md:5]: We need to change the version here 
```suggestion
## `6.6.0` -> `6.8.0`
```

**Review by stefaniacardenas — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kopijunkie) [reply CHANGELOG.md:61]: It looks like we might've lost this CHANGELOG not between releases. I couldn't determine which version it was released in from the Jira ticket based on the PR. @ethan605 any idea which release this is meant to be under?

Comment (ethan605) [reply CHANGELOG.md:61]: It should be from `6.7.0`, because the PR was merged on 19 Mar, and `6.6.0` was released on 11 Mar :)

Comment (kopijunkie) [reply CHANGELOG.md:61]: Thanks! Updated the Github release notes for `6.7.0` now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,27 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-version]
 
+## [6.8.0] - 2021-05-06
```
Comment (stefaniacardenas) [inline CHANGELOG.md:11]: The date should be updated
```suggestion
## [6.8.0] - 2021-05-13
```

**Review by stefaniacardenas — CHANGES_REQUESTED** [review]

**Review by stefaniacardenas — APPROVED** [review]

**Pull request merged**